src/sagemaker/huggingface/llm_utils.py

```diff
@@ -81,7 +81,8 @@ def get_huggingface_model_metadata(model_id: str, hf_hub_token: Optional[str] =
     Returns:
         dict: The model metadata retrieved with the HuggingFace API
     """
-
+    if len(model_id) == 0:
```

Comment:
nit: `if not model_id`
This is the best practice of checking strings in python.